src/classes/handlers/ContactTriggerHandler.cls

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by Mariya on 14.11.2022.
+ */
+
+public with sharing class ContactTriggerHandler {
+
+    public static void handler(List<Contact> newTrigger,  Map<Id,Contact> oldMapContacts, Map<Id,Contact> newMapContacts, System.TriggerOperation triggerOperation){
```

Comment:
MINOR
Review Code Convention and follow naming and styles from the article:
https://noltic.myjetbrains.com/youtrack/articles/UCT-A-23/Code-Convention

e.g.: 
List variable name must be in plural form - newContacts/newRecords
Map variable name must provide clarity on key-value stored, name in singular - Map<Id,Contact> contactById

add space after method signature, before body -  name(Id id) {}

Reply:
ok, thank you